app/templates/components/forms/user-profile-form.hbs

```diff
@@ -52,10 +52,24 @@
       </div>
       <div class="field">
         <label class="details">{{t 'Phone'}}</label>
-        <Input
-          @type="text"
-          @name="phone"
-          @value={{this.user.contact}} />
+        <div class="ui left action input">
+          <UiDropdown @class='ui selection dropdown'>
+            <i class="dropdown icon"></i>
+            <div class="default text">{{t 'Country Code'}}</div>
+            <div class="menu">
+              {{#each countries as |country|}}
+                <div class="item">
+                  <i class="{{lowercase country.code}} flag"></i>
+                  {{country.phone}}
```

Comment:
Show the country too, the user would want to search the code on the basis of the country name.